src/main/java/com/minecolonies/blocks/AbstractBlockMineColonies.java

```diff
@@ -0,0 +1,40 @@
+package com.minecolonies.blocks;

```

Comment:
Can we delete this class? Lets make an abstraction until we need to, this is something that would be easy to add in later if we needed it.